commands.h

```diff
@@ -1,83 +1,43 @@
-#ifndef COMMANDS_H
-#define COMMANDS_H
```

Comment:
Why remove the guard?

Reply:
It's only included in main. And since it's not a typical header file, I'm failing to see how this might get included multiple times.

Reply:
I just see it has standard practice that has no harm. If we are going strictly by need, then the same is true for all our header files. None of them include any others, so we could in theory remove guards from all of them

For practice, I think it would be reasonable for someone (like me) to include this file in their config.h so their linters can find the symbols.

So I see the guards as having some benefit for no cost.